test/e2e/tests/solana/common-solana.ts

```diff
@@ -1709,6 +1611,7 @@ export async function withSolanaAccountSnap(
       fixtures: fixtures.build(),
       title,
       dapp: true,
+      withSolanaWebSocket: true,
```

Comment:
we pass this to our `setupMocking` function, so it knows if we need to re-route all the Solana ws requests to our local websocket server